src/main/java/com/atlassian/bitbucket/jenkins/internal/model/deployment/BitbucketDeployment.java

```diff
@@ -0,0 +1,122 @@
+package com.atlassian.bitbucket.jenkins.internal.model.deployment;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * The details of a deployment.
+ *
+ * @since deployments
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class BitbucketDeployment {
+
+    private static final String DEPLOYMENT_SEQUENCE_NUMBER = "deploymentSequenceNumber";
+    private static final String DESCRIPTION = "description";
+    private static final String DISPLAY_NAME = "displayName";
+    private static final String ENVIRONMENT = "environment";
+    private static final String KEY = "key";
+    private static final String STATE = "state";
+    private static final String URL = "url";
+
+    private final long deploymentSequenceNumber;
```

Comment:
It's the smallest of nits, but Jenkins run numbers are always ints so the long precision isn't usable here